client/workitemtype.go

```diff
@@ -0,0 +1,36 @@
+package client
+
+import (
+	"fmt"
+	"golang.org/x/net/context"
+	"net/http"
+	"net/url"
+)
+
+// ShowWorkitemtypePath computes a request path to the show action of workitemtype.
+func ShowWorkitemtypePath(id string) string {
```

Comment:
Sure, but those are single words. We could make those cased as well. 

I'm just suspecting the goa goify type name method upper case the first letter only, which is why it works with login and version. But if the resource name has the proper case to begin with, hopefully, it will keep it. If not we have something to fix upstream :)